info/rules-mapping.md

```diff
@@ -0,0 +1,104 @@
+| Diktat Rule | Code Style |
+| ----------------------------------------- | ------ |
+| VARIABLE_NAME_INCORRECT | [1.1.1](diktat-coding-convention.md#1.1.1) |
+| VARIABLE_HAS_PREFIX | [1.1.1](diktat-coding-convention.md#1.1.1) |
+| IDENTIFIER_LENGTH | [1.1.1](diktat-coding-convention.md#1.1.1) |
+| GENERIC_NAME | [1.1.1](diktat-coding-convention.md#1.1.1) |
```

Comment:
I believe, those anchors are incorrect? We have something like `r1.1.1` in the actual document.

Reply:
thanks! fixed